### ISSUE TITLE
Configuration - Setting in config.xml not shown

### ISSUE THREAD

**Issue by nicolasbrossard** [aces/Loris#5169]
When a boolean setting is set in config.xml, it is not displayed on the front-end of the config module. For example, I had useProject set to true and none of the radio buttons ('Yes' or 'No') were checked in the front-end.

Found during testing of version 20.0 on IBIS stg.

Original Redmine ticket: https://redmine.cbrain.mcgill.ca/issues/15917

**Comment by ridz1208** [comment]
@nicolasbrossard could you test that out on 22 now ?
but with a different config setting then useProject cause that was removed recently

**Comment by nicolasbrossard** [comment]
@ridz1208 Here's what I did on the test VM database: I removed the record in table `Config` associated to the `useConsent` entry in `ConfigSettings`. Since there is a value for `useConsent` in `config.xml`, which is a boolean, it should be displayed on the front-end. It is, but it shows up as a text field, not a radio button with values true/false. So the behaviour is a bit different but it is still a bug.
